src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={tweetsArray[0]}></Tweet>
```

Comment:
Here, you could've mapped instead of passing it 3 times, like:

`{tweetsArray.map((el) =>{ return <Tweet tweet={el}/> })}`